README.md

```diff
@@ -1,114 +1,112 @@
-*Vulkan is not yet out for QuestCraft, please wait!*
+### Please do not enable seated mode in the in-game VR settings!
 
 ## Included Mods 
 
-**Did you experience a mod-related crash when loading MC? You can do 1 of these solutions to find the cause and fix it!**
-
-**Easy Way**                                                                                                                                                                                                                                                                                  
-Join the QuestCraft [Discord Server](https://discord.gg/questcraft) and go to #bot-commands. Type /logs and upload your log file (which can be retrieved from Android/data/com.qcxr.qcxr using a file viewer e.g Windows Explorer). The QuestCraft Bot, Crafty, will give you feedback and help you fix any issues.
+**Did you experience a mod-related crash when loading MC? It's (kind of) simple to fix it!**
+                                                                                                                                                                                                                                                                                  
+Join the QuestCraft Discord Server at **discord.gg/QuestCraft** and go to #bot-commands. Type /logs and upload your log file (which can be retrieved from Android/data/com.qcxr.qcxr using a file viewer (e.g. **Windows Explorer** or **[Android File Transfer for macOS](https://www.android.com/filetransfer/)**). The QuestCraft Bot, CraftyVR, will give you ways to fix the crash. If not, you can scroll to the bottom of the retrieved log in a text editor and check the Fabric error message for causes and possible solutions. 
 
-**Manual Way**                                                                                                                                                                                                                                                                                      
-To find out what caused the crash, you can open the latestlog.txt file. You can find it in Android/data/com.qcxr.qcxr with a file viewer. You can go to the bottom of the file and look for a Fabric error message. This message will tell you which mods caused the crash. Once you know what caused the problem, you can fix it.
 
-If you need help with these solutions, please go to the support channel of the QuestCraft [Discord](https://discord.gg/questcraft). 
+If you need help with crashes and other issues, please go to the support channel of the QuestCraft Discord (discord.gg/QuestCraft). 
 
-### Non-Performance Mods:
+### Standard Mods:
```

Comment:
core mods

also this is not a thorough list, whats the point?

Reply:
May you elaborate on this more?

Reply:
why include a list of pre included mods if its not complete and is subject to constant change?

Reply:
I'll maintain this list as it can be helpful for people who might accidentally install a duplicate or a mod that is incompatible with the mods here. And it will be very easy to maintain once I (eventually) get write access.

Reply:
Also, some mods aren't really a "core" mod. Fabric API and Vivecraft yes, but Simple Voice Chat and Mod Menu?